webview/src/shared/components/tooltip/Tooltip.tsx

```diff
@@ -6,6 +6,27 @@ import 'tippy.js/dist/tippy.css'
 export const HEADER_TOOLTIP_DELAY = 100
 export const CELL_TOOLTIP_DELAY = 1000
 
+const hideOnEsc = {
+  defaultValue: true,
+  fn({ hide }: { hide: () => void }) {
+    function onKeyDown(event: KeyboardEvent) {
```

Comment:
Things appear to be working correctly for the body cells but the `esc` key isn't working for the header cells on my end:

https://user-images.githubusercontent.com/43496356/189126147-69985ffe-bf05-4a68-8eb7-4f37e5071061.mov

Reply:
Yep, doesn't work... isn't obvious why either.

Reply:
I'll fix it today.